spec/system/publishers/publishers_can_manage_job_applications_for_a_vacancy_spec.rb

```diff
@@ -70,17 +70,6 @@
       expect(page).to have_content("Not Considering (3)")
     end
 
-    scenario "Changing a single status", :js do
```

Comment:
Removed this test as it was quite flaky and we already have a couple of tests asserting that we can change the status of an individual job application